nmdc_automation/re_iding/scripts/data/nmdc:sty-11-33fbta56/nmdc:sty-11-33fbta56_updated_record_identifiers.tsv

```diff
@@ -450,3 +450,2033 @@ nom_analysis_activity_set	nmdc:580d7176f039405da3342f8fe86c08bf	nmdc:wfnom-13-qh
 nom_analysis_activity_set	nmdc:54b56467b5164c2fb12c900aff7f4142	nmdc:wfnom-13-frt40706.1
 data_object_set	nmdc:014bfe902eeedfd2f1876782f516f0e3	nmdc:dobj-13-5rakzk42
 data_object_set	emsl:output_570845	nmdc:dobj-13-xw6d6f81
+nom_analysis_activity_set	nmdc:wfnom-13-k381q936.1	nmdc:wfnom-13-pgjtwe83.1

```

Comment:
i believe lines 453 to 502 should be deleted as this is a napa id to napa id mapping and the values in the third column don't exist.
example 
grep nmdc:wfnom-13-k381q936.1 *_updated_record_identifiers.tsv
nom_analysis_activity_set	nmdc:b712f0b5e8dc4c5394a5f70aaa755b4f	nmdc:wfnom-13-k381q936.1
nom_analysis_activity_set	nmdc:wfnom-13-k381q936.1	nmdc:wfnom-13-pgjtwe83.1
first line should be kept, that is a legacy to napa mapping
the second line is a napa to napa. nmdc:wfnom-13-k381q936.1 exists in prod mongo, nmdc:wfnom-13-pgjtwe83.1 does not

Reply:
please double check that for these rows the 'new id' doesn't exist.

Reply:
Query with the new_id values returns 0 results

Reply:
Query with the 50 C2 values returns 50 NOM workflow results

Reply:
https://github.com/microbiomedata/nmdc_automation/pull/187/commits/0c984ee2b1599397c00806ff0eff87b71b40eb28